### ISSUE TITLE
ModuleNotFoundError: No module named 'jwt' - Add to requirement.txt

### ISSUE THREAD

**Issue by harry3689** [bigeyex/python-adminui#38]
The python module _PyJWT_ has to be added to requirement.txt since the following error is produced while working with the source code in development mode:

```
(env) harry@harry-VirtualBox:~/test/python-adminui/python$ python example_table.py
Traceback (most recent call last):
  File "/home/harry/test/python-adminui/python/example_table.py", line 2, in <module>
    from adminui import *
  File "/home/harry/test/python-adminui/python/adminui/__init__.py", line 1, in <module>
    from .form import *
  File "/home/harry/test/python-adminui/python/adminui/form.py", line 3, in <module>
    from .app import callbackRegistry
  File "/home/harry/test/python-adminui/python/adminui/app.py", line 3, in <module>
    import jwt
ModuleNotFoundError: No module named 'jwt'
```

**Comment by harry3689** [comment]
After 1.2 update I believe it is appropriate to add aioFlask also to requirement.txt file as referenced by this issue : #26. This error shows up while using adminui after _pip install adminui_ _**(Not in development mode)**_

The console logs is shown below:

```
[2022-05-10 15:39:10,092] ERROR in app: Exception on /api/page_action [POST]
Traceback (most recent call last):
  File "/home/harry/env/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/harry/env/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/harry/env/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harry/env/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/harry/env/lib/python3.9/site-packages/flask/app.py", line 1606, in ensure_sync
    return self.async_to_sync(func)
  File "/home/harry/env/lib/python3.9/site-packages/flask/app.py", line 1627, in async_to_sync
    raise RuntimeError(
RuntimeError: Install Flask with the 'async' extra in order to use async views.
```

**Comment by bigeyex** [comment]
fixed in 1.2.2
